0003-interledger-protocol/0003-interledger-protocol.md

```diff
@@ -172,78 +131,66 @@ As with the [internet protocol](https://tools.ietf.org/html/rfc791#section-2.3),
 
 The interledger module translates interledger addresses to local ledger addresses. Connectors and local ledger interfaces are responsible for translating addresses into interledger routes and local routes, respectively.
 
-Addresses are hierarchically structured strings consisting of segments delimited by the period (`.`) character. In order to distinguish the present address format from future or alternative versions, the protocol prefix `ilp:` MUST be used:
+Addresses are hierarchically structured strings consisting of segments delimited by the period (`.`) character.
 
 ```
-ilp:us.bank1.bob
+g.us.bank1.bob
 ```
 
-Care must be taken in mapping interledger addresses to local ledger accounts. Examples of address mappings may be found in "Address Mappings" ((TODO)).
+More information about ILP addresses can be found in the [ILP Address Specification](../0015-ilp-addresses/).
+
+The mapping from addresses to local accounts on a ledger is defined by the ledger protocol.
 
 ### Connectors
 
 Connectors implement the interledger protocol to forward payments between ledgers. Connectors also implement the [Connector to Connector Protocol (CCP)](../0010-connector-to-connector-protocol/) to coordinate routing and other interledger control information.
```

Comment:
connectors also implement (or include a module that implements) the ledger protocol of the ledgers where they have accounts